specifications/api/swagger.yaml

```diff
@@ -1363,7 +1363,7 @@ x-transactionTTL: &transactionTTL
 x-unspecified: &unspecified
   type: string
   enum:
-    - uspecified
+    - unspecified
```

Comment:
@paweljakubas we should remove this "unspecified" and use null/undefined/missing to denote unspecified.

Reply:
Yes, we've discussed this with Paweł and for now just added some tests (also covering other "failing" scenarios) and make them "pending"), such that then can be enabled later once the issues are fixed. Findings also listed here -> https://jira.iohk.io/browse/ADP-909.